python/ray/data/_internal/plan.py

```diff
@@ -250,21 +244,20 @@ def get_plan_as_string(self, classname: str) -> str:
         SCHEMA_LINE_CHAR_LIMIT = 80
         MIN_FIELD_LENGTH = 10
         INDENT_STR = " " * 3
-        trailing_space = " " * (max(num_stages, 0) * 3)
```

Comment:
Previously we assume that there's only one input, but there can be many (union, zip). For this pr we keep that assumption, but we just append it to the single input to the end of the string, instead of formatting it correctly with the number of tabs.